utils/resultstable_test.go

```diff
@@ -447,7 +447,7 @@ func TestGetApplicableCveValue(t *testing.T) {
 				EntitledForJas: true,
 			},
 			cves:           nil,
-			expectedResult: ApplicabilityUndetermined,
+			expectedResult: NotScanned,
```

Comment:
How it is `NotScanned` with JAS? before the changes it was `ApplicabilityUndetermined`, so it should be the new default when JAS is enabled `NotCovered`.

In addition I would suggest to add names to the tests to be clear what it tests.